mfrc522/mfrc522.py

```diff
@@ -0,0 +1,206 @@
+from enum import Enum
+
+class NoTagError(Exception):
+    pass
+
+class TransmissionError(Exception):
+    pass
+
+class MFRC522:
+
+    MAX_LEN = 16
+
+    class Commands(Enum):
+        PCD_IDLE            = 0x00
+        PCD_AUTHENT         = 0x0E
+        PCD_RECEIVE         = 0x08
+        PCD_TRANSMIT        = 0x04
+        PCD_TRANSCEIVE      = 0x0C
+        PCD_RESETPHASE      = 0x0F
+        PCD_CALCCRC         = 0x03
+
+    class Registers(Enum):
```

Comment:
Those are constants, yet names are in CamelCase so that they are consistent with the mfrc522 datasheet.